great_expectations/core/expectation_suite.py

```diff
@@ -272,7 +273,10 @@ def _is_fresh(self) -> ExpectationSuiteFreshnessDiagnostics:
         try:
             key = self._store.get_key(name=self.name, id=self.id)
             suite_dict = self._store.get(key=key)
-        except StoreBackendError:
+        except (
+            StoreBackendError,  # Generic error from stores
+            InvalidKeyError,  # Ephemeral context error
```

Comment:
Should have a ticket in for this? Looks like a broken abstraction if we need to know about ephemeral contexts specifically here. Does this just get resolved in the fabled stores refactor?

Reply:
Ugh yeah this is a weird deviation - it will get resolved whenever we refactor stores sure but perhaps we should update this store to be consistent